modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path

```diff
@@ -370,6 +370,11 @@
 		<td>//div[(@class='table-list-title') and contains (.,'${key_name}')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>SIDEBAR_ADVANCED_TAB</td>
+		<td>//div[contains(@class,'lfr-objects')]//button[contains(text(),'Advanced')]</td>
+		<td></td>
+	</tr>
```

Comment:
There are other paths for that, but they are from other components, such as Forms and Documents and Media.

So I think it's good to have one for Objects to prevent changes to other components' paths from affecting this test.

Reply:
That's fine. I'm surprised there isn't a generic path for this but if the paths are different for each component, it makes sense.